docs/UserGuide.md

```diff
@@ -147,36 +147,92 @@ Examples:
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang` e.g. `alex yeoh` will return `alextan@example.com`, `berniceyeoh@example.com`.
-* For finding by Salary, command will find employees whose salaries fall within the range of LOWERBOUND to UPPERBOUND inclusive of LOWERBOUND and UPPERBOUND.
-* The LOWERBOUND **cannot be larger than** the UPPERBOUND. 
-* There must be a **spacing before and after the dash**
-* LOWERBOUND and UPPERBOUND must be **non-negative integers** and **cannot be larger than 1 000 000 000**. 
+* Finds employees whose attributes satisfy all the requirements of the FindCommand
+
+Examples:
+* `find i/A00 n/alex p/9001` returns employees whose ID contains `A00`, whose name contains the word `alex` and whose phone number contains `9001`.
+
+#### Finding an employee by name: `find n/`
+
+Find employees whose name matches at least one of the keywords in full.
+
+Format: `find n/KEYWORD [OTHER KEYWORD]`
+
+* The order of the keywords does not matter.
+* Only **full keywords** will be matched.
+* Employees whose name **matches at least one keyword** will be returned.
 
 Examples:
-* `find i/A00` returns employees with IDs `A00`, `A00001`, `a0001`.
-* `find i/A00 p/9001` returns employees who match the ID and also have Phone numbers such as `90018989`, `9001`.
 * `find n/John` return employees with names `john` and `John Doe`.
 * `find n/alex david` returns employees with names `Alex Yeoh`, `David Li`.
-* `find d/Finance` return employees in departments `finance`, `Finance 1`.
-* `find d/finance marketing` returns employees in departments `finance 1`, `marketing 2`.
-* `find r/Manager` returns employees whose roles are `manager`, `Executive Manager`.
-* `find r/executive manager` returns employees whose roles are `Executive Chef`, `Product manager`.
+
+#### Finding an employee by department: `find d/`
+
+Find employees whose department matches at least one of the keywords in full.
+
+Format: `find d/KEYWORD [OTHER KEYWORD]`
+
+* Works the same as finding an employee by name.
+
+#### Finding an employee by role: `find r/`
+
+Find employees whose role matches at least one of the keywords in full.
+
+Format: `find r/KEYWORD [OTHER KEYWORD]`
+
+* Works the same as finding an employee by name.
+
+#### Finding an employee by email: `find e/`
+
+Find employees whose email contains at least one of the keywords.
+
+Format: `find e/KEYWORD [OTHER KEYWORD]`
+
+* The order of the keywords does not matter.
+* Email will be matched if it **contains a keyword**.
+* Email **containing at least one keyword** will be returned.
+
+Examples:
 * `find e/alex bernice` returns employees with emails `alexyeoh@example.com`, `bernicetan@example.com`.
+
+#### Finding an employee by ID: `find i/`
+
+Finds employees whose ID contains the keyword.
+
+Format: `find i/KEYWORD`
+
+* The command will find employees whose ID and Phone number contains the respective keywords.
```

Comment:
Is this supposed to be just ID, and not "ID and Phone number"? (and also "respective")

Reply:
Oh yes you're right! Thanks so much for spotting this mistake